ghcide/src/Development/IDE/Core/Compile.hs

```diff
@@ -1026,7 +1026,7 @@ getDocsBatch hsc_env _mod _names = do
               pure . (name,) $
                 if isNothing mb_doc_hdr && Map.null dmap && Map.null amap
                 then Left $ NoDocsInIface mod $ compiled name
-                else Right (Map.lookup name dmap, Map.findWithDefault mempty name amap)
+                else Right (Map.lookup name dmap, Map.lookup name amap)
```

Comment:
There was arg doc stubbing.

The idiomatic way is to use `Maybe`-alike there & `Map.lookup` already does `Maybe`, so provisioning it.